sdk/storage/azure-storage-file/src/main/java/com/azure/storage/file/implementation/AzureFileStorageBuilder.java

```diff
@@ -29,6 +29,22 @@ public AzureFileStorageBuilder version(String version) {
         return this;
     }
 
+    /*
+     * Only update is supported: - Update: Writes the bytes downloaded from the source url into the specified range.
+     */
```

Comment:
Why is this on the builder. This seems like a parameter that should be passed into an api

Reply:
@tg-msft Do you have more insight on what this gets used for?

Reply:
It came with the new swagger:

```json
        "FileRangeWriteFromUrl": {
            "name": "x-ms-write",
            "x-ms-client-name": "fileRangeWriteFromUrl",
            "in": "header",
            "description": "Only update is supported: - Update: Writes the bytes downloaded from the source url into the specified range.",
            "required": true,
            "type": "string",
            "enum": [
                "update"
            ],
            "default": "update",
            "x-ms-enum": {
                "name": "FileRangeWriteFromUrlType",
                "modelAsString": false
            }
        },
```

Reply:
I am pretty sure that comes from [this api](https://docs.microsoft.com/en-us/rest/api/storageservices/put-range). In normal write operations, it's used to specify whether the operation is to update the data or clear the range. When specifying a source url (hence the "fromURL", the only viable option is update.

This should just be an internal constant that is used for the writeRangeFromURL method.